<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/window/assets/emby.png</normaloff>:/window/assets/emby.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>540</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Plain</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::EditTrigger::SelectedClicked</set>
    </property>
    <property name="showDropIndicator" stdset="0">
     <bool>false</bool>
    </property>
    <property name="dragDropOverwriteMode">
     <bool>false</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Datei</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Hilfe</string>
    </property>
    <addaction name="actionAboutEmbyExplorer"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>28</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/tb/assets/toolbar_preferences.svg</normaloff>:/tb/assets/toolbar_preferences.svg</iconset>
   </property>
   <property name="text">
    <string>Einstellungen</string>
   </property>
   <property name="toolTip">
    <string>Einstellungen</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/tb/assets/toolbar_quit.svg</normaloff>:/tb/assets/toolbar_quit.svg</iconset>
   </property>
   <property name="text">
    <string>Beenden</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAboutEmbyExplorer">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/tb/assets/toolbar_about.svg</normaloff>:/tb/assets/toolbar_about.svg</iconset>
   </property>
   <property name="text">
    <string>Über Emby Explorer</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/tb/assets/toolbar_about.svg</normaloff>:/tb/assets/toolbar_about.svg</iconset>
   </property>
   <property name="text">
    <string>Über Qt</string>
   </property>
  </action>
  <action name="actionAuthenticate">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/tb/assets/toolbar_login.svg</normaloff>:/tb/assets/toolbar_login.svg</iconset>
   </property>
   <property name="text">
    <string>Anmelden</string>
   </property>
   <property name="toolTip">
    <string>Am Emby Server anmelden</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionFetch">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/tb/assets/toolbar_fetch.svg</normaloff>:/tb/assets/toolbar_fetch.svg</iconset>
   </property>
   <property name="text">
    <string>Abrufen</string>
   </property>
   <property name="toolTip">
    <string>Daten vom Server abrufen</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionDetails">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/tb/assets/toolbar_details.svg</normaloff>:/tb/assets/toolbar_details.svg</iconset>
   </property>
   <property name="text">
    <string>Details</string>
   </property>
   <property name="toolTip">
    <string>Details anzeigen</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionExport">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/tb/assets/toolbar_export.svg</normaloff>:/tb/assets/toolbar_export.svg</iconset>
   </property>
   <property name="text">
    <string>Export</string>
   </property>
   <property name="toolTip">
    <string>Export in Excel Datei</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
